src/api/spec/services/scm_status_reporter_spec.rb

```diff
@@ -69,5 +69,63 @@
 
       it { expect { subject }.to change(EventSubscription, :count).by(-1) }
     end
+
+    context 'when sending a report back to GitHub' do
+      let(:event_payload) do
+        { project: 'home:danidoni', package: 'hello_world',
+          repository: 'openSUSE_Tumbleweed', arch: 'x86_64' }
+      end
+      let(:event_subscription_payload) do
+        { scm: 'github', short_package_name: 'hello_world-1234567',
+          target_repository_full_name: 'danidoni/hello_world', commit_sha: '1234567' }
```

Comment:
```suggestion
          target_repository_full_name: 'danidoni/hello_world', commit_sha: '123456789' }
```

It doesn't make any difference in the spec, but can be confusing to have a value like the short commit sha when the `commit_sha` is always stored in the long version.